src/components/app/pageEvents/components/stateEventsDialogContent.tsx

```diff
@@ -14,10 +16,12 @@ interface StateEventsDialogProps {
 }
 
 export function StateEventsDialogContent({ state, events }: StateEventsDialogProps) {
+  usePreventOverscroll()
+
   const parsedState = state.toUpperCase() as keyof typeof US_MAIN_STATE_CODE_TO_DISPLAY_NAME_MAP
-  const stateEvents = events?.filter(
-    event => event.data.state?.toLowerCase() === state.toLowerCase(),
-  )
+  const stateEvents = events
+    ? events?.filter(event => event.data.state?.toLowerCase() === state.toLowerCase())
+    : []
```

Comment:
This should do the trick.

```suggestion
  const stateEvents = events?.filter(event => event.data.state?.toLowerCase() === state.toLowerCase()) ?? []
```